src/api/controller.js

```diff
@@ -2087,6 +2087,10 @@ export default class ApiController {
    */
   async getRelatedPosts(req, res) {
     function formatArray(array) {
+      if (!array || !array.length) {
+        return '(NULL)';
+      }
+
```

Comment:
wouldn't it be better to avoid doing relevant query part in the first place?
I mean: let's fix the cause, not the consequence